resources/css/database.css

```diff
@@ -0,0 +1,63 @@
+body{
+	padding: 10px 30px;
+}
+
+.pagetitle {
```

Comment:
CSS classes should be made with camel case. This one should be "pageTitle" instead. All of the classes below should be changed to match this style (both in this file and in the database.blade.php file!)